### PR TITLE
fix(relatedItem) Modified related item widget to indicate possible update items or take items.

### PR DESCRIPTION
**Pull request by mdhaman**
- Adding an indicator to related items for 'associate as update' or 'associated as take'

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: scripts/superdesk-archive/index.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+//styles
+import './styles/related-item.less';
```
Comment (gbbr) [inline scripts/superdesk-archive/index.js]: Would it serve better de-coupling if `relative-item-widget` would import its own styles?

**Comment by gbbr** [comment]
LGTM besides minor structural comment

File: scripts/superdesk-archive/index.js (hunk 1 above)
Comment (mdhaman) [inline scripts/superdesk-archive/index.js]: sure will do that.

**Pull request merged**